playground/src/components/Toolbar.tsx

```diff
@@ -1,10 +1,19 @@
 import { HStack } from '@/styled-system/jsx'
 import { PropsWithChildren } from 'react'
 import { Logo } from './Logo'
+import { css } from '@/styled-system/css'
+import { version } from 'package.json'
```

Comment:
This is pointing to version of the playground, not panda's version.

We need to update it to something like @pandacss/dev/package.json